.github/review-bot.yml

```
@@ -9,24 +9,20 @@ rules:
   - name: Relay and system files
     condition:
       include:
-        - ^relay\/kusama\/.*
-        - ^relay\/polkadot\/.*
+        - ^relay\/.*
         - ^system-parachains\/.*
-        - ^CHANGELOG$
-      exclude: 
-        - ^relay\/.+\.adoc$
+        - ^CHANGELOG.md$
```

The `CHANGELOG` file does not have a file type.

```suggestion
        - ^CHANGELOG$
```

---

https://github.com/polkadot-fellows/runtimes/pull/28/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed I changed it here

---

Ah, I didn't see it. I was comparing it against `main`. 

Thanks for clarifying it!